engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuKafkaRuntimeTestMixin.scala

```diff
@@ -0,0 +1,59 @@
+package pl.touk.nussknacker.engine.lite.kafka
+
+import org.apache.commons.io.FileUtils
+import org.apache.kafka.clients.admin.NewTopic
+import org.scalatest.TestSuite
+import pl.touk.nussknacker.engine.build.EspProcessBuilder
+import pl.touk.nussknacker.engine.canonize.ProcessCanonizer
+import pl.touk.nussknacker.engine.graph.EspProcess
+import pl.touk.nussknacker.engine.kafka.{KafkaClient, KafkaUtils}
+import pl.touk.nussknacker.engine.marshall.ProcessMarshaller
+import pl.touk.nussknacker.engine.spel.Implicits._
+
+import scala.collection.JavaConverters._
+import java.io.File
+import java.nio.charset.StandardCharsets
+import scala.compat.java8.OptionConverters.RichOptionForJava8
+
+trait NuKafkaRuntimeTestMixin { self: TestSuite =>
+
+  protected def kafkaBoostrapServer: String
+
+  protected def prepareTestCaseFixture(testCaseName: String, prepareScenario: (String, String) => EspProcess): NuKafkaRuntimeTestTestCaseFixture = {
+    val rootName = self.suiteName + "-" + testCaseName
+    val inputTopic = rootName + "-input"
+    val outputTopic = rootName + "-output"
+    KafkaUtils.usingAdminClient(kafkaBoostrapServer) { client =>
```

Comment:
let's add TODO: to replace it with kafkaClient when we get rid of zkaddress?

Reply:
:+1: added